### DISCUSSION TITLE
How can i  add nested navigation menu in left_drawer?

### DISCUSSION THREAD

**Discussion by nicedouble** [zauberzeug/nicegui#1299] [Q&A]
### Description

I am new to nicegui,and it's totally amazing!
I want to add a navigation menu in left_drawer(or nested menu),how can i achieve it?
in Qussar,i find the similar layout: [https://quasar.dev/layout/drawer#example--with-navigation-menu](url)

**Comment by rodja** [comment]
Great you like this framework. Have you seen https://nicegui.io/documentation#page_layout?

**Reply by nicedouble** [reply]
Thanks for your reply.Actually i want to create a navigation menu in left_drawer,in Quasar,i found use q-list and q-item to achieve this.
![1691053369434](https://github.com/zauberzeug/nicegui/assets/40421045/10560c47-150c-4d33-94d7-30b61b64b397)

**Reply by v479038280** [reply]
Is there element like q-list and q-item  in nicegui?

**Reply by falkoschindler** [reply]
Not explicitly. But you can use q-list and q-item nonetheless: https://github.com/zauberzeug/nicegui/discussions/512#discussioncomment-5314453
